src/platforms/common/configuration/sampling.mdx

```diff
@@ -14,13 +14,18 @@ supported:
   - react-native
   - android
   - dotnet
+  - apple
 notSupported:
   - javascript.cordova
   - capacitor
 redirect_from:
   - /performance/sampling/
 ---
 
+<PlatformSection supported={["apple"]}>
+**Attention: This is a preview API available since 7.0.0-alpha.2. This API is not in stable state yet. It may change in the future.**
+</PlatformSection>
```

Comment:
```suggestion
<PlatformSection supported={["apple"]}>

<Note>

Attention: This is a preview API available since 7.0.0-alpha.2. This API is not in stable state yet. It may change in the future.

</Note>

</PlatformSection>
```